sdk/websites/Azure.ResourceManager.AppService/src/autorest.md

```diff
@@ -849,4 +849,40 @@ directive:
             "Aborted",
             "Running"
         ]
+  - from: WebApps.json
+    where: $.paths['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/hybridConnectionRelays'].get
+    transform: >
+        $['operationId'] = "WebApps_GetAllTheHybridConnectionData";
```

Comment:
don't change the operationId

Reply:
yeah we should not change the operation id.
If we want to change the generated method name, we should use our configurations

Reply:
use this configuration to change the name of a method:
override-operation-name:
  operationId: NewName

Reply:
fixed